integration-tests/tests/src/tests/sync/mixed.ts

```diff
@@ -211,13 +211,29 @@ function performTests(flexibleSync: boolean) {
 }
 
 describe.skipIf(environment.missingServer, "mixed", () => {
+  // When running on CI we connect through mongodb-atlas instead of local-mongodb
+  const { mongodbClusterName } = environment;
+  const clusterReplacement =
+    typeof mongodbClusterName === "string"
+      ? {
+          "services/mongodb/config.json": {
+            type: "mongodb-atlas",
+            config: {
+              clusterName: mongodbClusterName,
+              readPreference: "primary",
+              wireProtocolEnabled: false,
+            },
+          },
+        }
+      : undefined;
```

Comment:
Would it make sense to move this to somewhere reusable? I expect we would want to run the flexible sync tests against realm-dev, for example, and it looks like it would need the same replacement logic.

Could it be inside `importApp` for example?

Reply:
I moved this to a `getDefaultReplacements` in the `import-app.ts`.

Reply:
Great!